keyboards/reviung/reviung46/readme.md

```diff
@@ -0,0 +1,27 @@
+# reviung46
+
+![reviung46](https://github.com/gtips/reviung/blob/master/reviung46/image/reviung46-01.jpg) 
```

Comment:
Thanks.
I designed the PCB so that multiple key layouts can work with one layout keyboard.json.
I have confirmed that it works properly on the test board.
The PCB design is similar to REVIUNG53.
https://github.com/gtips/reviung/tree/master/reviung53

Reply:
PCB data added.
Request review.
https://github.com/gtips/reviung/tree/master/reviung46

Reply:
> I designed the PCB so that multiple key layouts can work with one layout keyboard.json.

Incorrect. Your PCB clearly supports multiple layouts. The expectation is that you add multiple layout macros to support these.

> Request review.

There is nothing to review as you have not implement the extra layouts in `keyboard.json`.

Reply:
I’m so sorry.
It was my lack of understanding.
The last time I made a keyboard was three years ago, so I mixed up the way I did it three years ago.

I've fixed keyboard.json so I'm requesting a review.